Pirat/Program.cs

```diff
@@ -35,7 +41,10 @@ public static void CheckEnvironmentVariables()
                 
                 "PIRAT_ADMIN_KEY",
                 "PIRAT_SWAGGER_PREFIX_PATH",
-                "PIRAT_WEBAPP_ENVIRONMENT"
+                "PIRAT_WEBAPP_ENVIRONMENT",
+
+                "PIRAT_INIT_DB_TABLES_IF_NOT_EXIST",
+                "PIRAT_INIT_DUMMY_DATA_IF_NOT_EXIST"
```

Comment:
The variables should be optional with `false` being the default. Please also don't forget to edit the README.

Reply:
I do not see a disadvantage here to let them non optional. At least the developer or the server must initialize the fields with something. It's not much work...